### PR TITLE
Describe translation process

### PR DESCRIPTION
**Pull request by jlaur**
See also: https://github.com/openhab/openhab-addons/blob/main/README.md#translations

### CODE REVIEW

File: developers/utils/i18n.md [hunk 1]
```diff
@@ -475,3 +475,17 @@ List<ThingType> thingTypes = thingTypeRegistry.getThingTypes(Locale.GERMAN);
...
+## Managing translations
+
+Localization is managed with the cloud service [Crowdin](https://crowdin.com/) where translators can contribute translations.
+
+This is the process:
+
+- Texts in the original language (English) are provided (with the corresponding code) in pull requests.
+- When a pull request is merged, the texts will be automatically imported into Crowdin.
+- Texts can now be translated into multiple languages using Crowdin.
+- Once a day a pull request with updated translations is automatically created. Please note that all texts must be translated and approved in order to be included.
```
Comment (Confectrician) [inline developers/utils/i18n.md:488]:
```suggestion
- Once a day a pull request with updated translations is automatically created.
Please note that all texts must be translated and approved in order to be included.
```

**Review by Confectrician — COMMENTED** [review]
Thanks. 👍🏼

Just one small comment.
The website build will take care of a proper presentation on the page.
So we can also use our one line per sentence rule for ordered lists.

File: developers/utils/i18n.md (hunk 1 above)
Comment (jlaur) [reply developers/utils/i18n.md]: Thanks, I didn't know this would work inside an ordered list.

**Review by Confectrician — APPROVED** [review]
Thanks. 👍🏼

**Comment by fwolter** [comment]
Should we also mention the Maven commands to generate the default properties files, which are mandatory to make Crowdin work?

**Comment by jlaur** [comment]
> Should we also mention the Maven commands to generate the default properties files, which are mandatory to make Crowdin work?

Certainly. I should have marked the PR a draft to enforce some review time. I'll create a new PR tomorrow and do that. :)

**Comment by jlaur** [comment]
> > Should we also mention the Maven commands to generate the default properties files, which are mandatory to make Crowdin work?
> 
> Certainly. I should have marked the PR a draft to enforce some review time. I'll create a new PR tomorrow and do that. :)

@fwolter - see #1833.

**Comment by Confectrician** [comment]
> I should have marked the PR a draft to enforce some review time.

I am a simple man.
I see a good pr without draft note.
I presse Squash and merge.

Simple as that. 😁

**Pull request merged**